stdlib/public/core/Optional.swift

```diff
@@ -130,11 +130,40 @@ public enum Optional<Wrapped>: ExpressibleByNilLiteral {
 
   /// The presence of a value, stored as `Wrapped`.
   case some(Wrapped)
+}
+
+extension Optional: Sendable where Wrapped: ~Copyable & Sendable { }
+
+extension Optional: _BitwiseCopyable where Wrapped: _BitwiseCopyable { }
+
+extension Optional: Copyable /* where Wrapped: Copyable */ {}
```

Comment:
I would prefer the copyable extension to have higher priority than the sendable one (aka immediately after the optional declaration). Also same question with MemoryLayout, are we committing with the defaults? When are we adding the explicit?

Reply:
Yep, `Copyable` should come first!